docs/source/api.rst

```diff
@@ -34,10 +45,121 @@ Access the API using curl
 
     curl -X GET -H 'Authorization: Token <YOUR TOKEN>' https://public.vulnerablecode.io/api/
```

Comment:
IMHO this should be part of the examples, as an alternative to using Python. Here this is not the best place.